src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -316,26 +345,27 @@ class ReportActionItemMessageEdit extends React.Component {
                         />
                     </View>
 
+                    <View style={styles.alignSelfEnd}>
+                        <Tooltip text={this.props.translate('common.saveChanges')}>
+                            <TouchableOpacity
```

Comment:
AFAIK `TouchableOpacity` is being deprecated. Can we use `Pressable` instead?

Reply:
Touchable Opacity was used to match exactly the send button we have in the `ReportActionComposer.js`.

In my opinion, changing it to Pressable would also mean changing it in ReportActionComposer which is already out of the scope of this PR.

Reply:
I still think that we should change this to a `Pressable` since `TouchableOpacity` is being deprecated. I understand if we wanna tackle `ReportActionComposer` in a followup so we don't add more testing scope to this issue though.

Reply:
If there's going to be a follow up PR changing ReportActionComposer to use Pressable instead of TouchableOpacity then it makes more sense to me to do both at the same time.

By the way, TouchableOpacity is not deprecated. On reactnative.dev the recommend to use Pressable if you want to have more control:
> If you're looking for a more extensive and future-proof way to handle touch-based input, check out the Pressable API.

Also, Pressable is not the same as TouchableOpacity since the later implements animation out of the box, Pressable doesn't have animation.
Pressabe is designed to be a building block for custom buttons. So the ultimate approach would be to implement out own Touchable Opacity using Pressabe.